code-challenges.js

```diff
@@ -39,7 +82,17 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+// declare a function that accepts a string as an argument
+const noVowelsVer1 = (string) => {
+  // first split string into array and use a filter to go through all letters in the array
+  return string.split("").filter(value => {
+    // return array with the condition that it will only keep consonants--NO VOWELS (a, e, i, o u (sorry 'y'))
+    // join all the values of array to form the word
+    return value !== "a" && value !== "A" && value !== "e" && value !== "E" && value !== "i" && value !== "I" && value !== "o" && value !== "O" && value !== "u" && value !== "U"}).join("")
```

Comment:
This is great and works well! I was mentioning this to Diem as well, since she had a similar solution, but consider this: 

To make our code a bit more DRY, we could use:

`value.toLowerCase() !== 'a'`

This will make sure that we don't need to check for both uppercase and lower case.
To take it a bit further, we could delcare value to be lowercase way ahead of time, before your return, to make it even MORE DRY.

`value = value.toLowerCase() `
`return value !== 'a' && value !== 'e' ...`

Just something to think about!